testData/solutions/win32.json5

```diff
@@ -2609,495 +2834,13473 @@
+                    },
+                    "Options.TouchTypingEcho": {
+                        "schema": {
+                            "title": "Touch Typing Echo",
+                            "description": "Whether/how to echo text entered using the touch keyboard.",
+                            "enum": [
+                                0,
+                                1,
+                                2,
+                                3
+                            ],
+                            "enumLabels": [
+                                "None",
+                                "Characters",
+                                "Words",
+                                "Both characters and words"
+                            ],
+                            "default": 3
+                        }
+                    },
+                    "Options.TouchKeyboardNotification": {
+                        "schema": {
+                            "title": "Touch Keyboard Notification",
+                            "description": "Whether/how to notify when the touch keyboard appears/disappears.",
+                            "enum": [
+                                0,
+                                1,
+                                2
+                            ],
+                            "enumLabels": [
+                                "By Message",
+                                "By Sound",
+                                "By both Message and Sound"
+                            ],
+                            "default": 2
+                        }
+                    },
+                    "Options.TouchKeyboardChildPanelNotification": {
+                        "schema": {
+                            "title": "Touch Alternative Character Panel Popup Notification",
+                            "description": "How to announce the appearance of a panel of alternate keys in the touch keyboard.",
+                            "enum": [
+                                0,
+                                1
+                            ],
+                            "enumLabels": [
+                                "By both Message and Sound",
+                                "By Sound only"
+                            ],
+                            "default": 0
+                        }
+                    },
+
+                    // -------------- Custom Highligth -------------
+
+                    // -------- TODO: Check how to access ----------
+
+                    // ---------------- Research It ----------------
+
+                    "Research It Options.Primary": {
+                        "schema": {
+                            "title": "Primary Research Option",
+                            "description": "Defines the primary research option.",
+                            "type": "string"
+                        }
+                    },
+                    "Research It Exclusions.Exclude1": {
```

Comment:
What do the numbers mean in the keys?  Can there only ever be this many exclusions? Also, no defaults and no indication of what's being excluded.

Reply:
commented out all of these.